AnthonyGW/Bootcamp-XX-Flask-API
# Development, Files Cleanup

#### What does this PR do?
Implements RESTful workflow for the API's endpoints.
Adds details of features to the README
#### How should this be manually tested?
Register a new user and login on an active server using Postman.
Add the generated token to a header with prefix "Bearer " for authentication. The header should be named 'Authorization'.
#### Any background context you want to provide?
The project requires several more features which may be committed before the merge.
These are:
- Searching
- Pagination with limits
